main.cs

```diff
@@ -746,14 +745,13 @@ public enum CustomRoles
     Opportunist,
     Pelican,
     Pestilence,
-    Specter,
```

Comment:
Specter missed

Reply:
@Tommy-XL, thanks for pointing out that Spectre was missed. I'll correct that immediately.